src/docs/sphinx/tutorials/step12/Tutorial.rst

```diff
@@ -0,0 +1,357 @@
+| :math:`\phi`     | Porosity                | [-]              | 0.3                |
++------------------+-------------------------+------------------+--------------------+
+
+
+Inspecting results
+---------------------------------
+
+As defined in the ``Events`` section, we run this simulation for 497640 seconds. In the above examples, we requested silo-format output files. We can therefore import these into VisIt and use python scripts to visualize the outcome. 
+
+Using the poroelastic solver, Fig.3 shows the prediction of pore pressure distribution upon fluid injection.
+
+.. _problemVerificationFig1:
+.. figure:: PoroElastic_PP.png
+   :align: center
+   :width: 1000
+   :figclass: align-center
+
+   Fig.3 Simulation result of pore pressure distribution
+
+
+For the above poroelastic example, an analytical solution `(Detournay and Cheng, 1993)  <https://www.sciencedirect.com/science/article/pii/B9780080406152500113>`__ is hereby employed to verify the accuracy of the numerical results. Fig.4 shows the comparisons between the numerical predictions (marks) and the corresponding analytical solutions (solid curves) with respect to the distributions of pore pressure, radial displacement, effective radial and tangential stresses along the minimum horizontal stress direction (x-axis). One can observe that GEOSX results correlate very well with the analytical solutions for the poroelastic case. 
+
+
+.. _problemVerificationFig2:
+.. figure:: PoroElastic_X.png
+   :align: center
+   :width: 1000
+   :figclass: align-center
+
+   Fig.4 Comparing GEOSX results with analytical solutions
+
+
+
+For the same 3D wellbore problem, the poroplastic case is thereafter tested and compared with the poroelastic one. Fig.5 shows the distribution of :math:`\Sigma_yy` in the near wellbore region for both cases. As observed, a relaxation of the tangential stress along the direction of minimum horizontal stress is detected for the poroplastic case, which can be attributed to the plastic response of the rock.
+
+
+.. _problemVerificationFig3:
+.. figure:: PoroDP_Stress.png
+   :align: center
+   :width: 1000
+   :figclass: align-center
+
+   Fig.5 Simulation result of Syy: PoroElastic vs. PoroPlastic
+
+
+By using python scripts, we can extract the simulation results along any direction and provide detailed comparisons between different cases. Here, the pore pressure, radial displacement, radial and tangential effective stresses along the direction of minimum horizontal stress are obtained at different time steps and plotted against the corresponding ones of the poroelastic case. Because of fluid diffusion and coupling effect, Fig.6 shows that these solutions evolve with time for both cases. As mentioned above, plastic zone is developed in the vicinity of the wellbore, due to stress concentration. As for the far field region, these two cases become almost identical, as the rock deformation is governed by poroelasticity.    
```

Comment:
```suggestion
By using python scripts, we can extract the simulation results along any direction and provide detailed comparisons between different cases. Here, the pore pressure, radial displacement, radial and tangential effective stresses along the direction of minimum horizontal stress are obtained at different time steps and plotted against the corresponding ones of the poroelastic case. Because of fluid diffusion and coupling effect, Fig.6 shows that these solutions evolve with time for both cases. As mentioned above, a plastic zone is developed in the vicinity of the wellbore, due to stress concentration. As for the far field region, these two cases become almost identical, with the rock deformation governed by poroelasticity.    
```